master/reference/repo-structure.md

```diff
@@ -1,69 +1,63 @@
-   separate process.  It is installed directly into the `calico/node` image.
-
- - [libcalico](https://github.com/projectcalico/libcalico): Contains a variety
-   of helper methods and classes for manipulating the Calico data in an etcd
-   datastore, IP address management and useful namespace utilities to
-   manipulating container interfaces.  The libcalico library is used by
-   `calicoctl`.  It is imported as a standard Python library into the
-   `calicoctl` Python codebase.  libcalico also contains the `calico/test` Docker
-   image used for running both UTs and STs testing Calico with Docker.
-
-## Calico Orchestrator Integration Plugins
+## Core Components
+
+### [Felix](https://github.com/projectcalico/felix)
+
+| Artifact | Type | Description |
+|---------|-------|-----------|
+| `felix` | Binary |The heart of Calico networking, Felix interfaces with the Linux kernel to configure routes and ACLs that control network policy connectivity. Felix is a common component run in every orchestration system.|
+
+
+### [Calico-Containers](https://github.com/projectcalico/calico-containers)
+
+| Artifact | Type | Description |
+|---------|-------|-----------|
+| `calico/node`| Docker Image |  The Docker image used to run the Calico Felix agent (responsible for IP routing and ACL programming) and a BIRD BGP agent (for distributing routes between hosts) for Containerized workloads (i.e. not VMs with Openstack).  See [`calico_node` directory]({{site.baseurl}}/{{page.version}}/reference/architecture) for details. |
+| `calicoctl`| Binary | The command line tool used for starting a `calico/node`  container, configuring Calico policy, adding and removing containers in a  Calico network via orchestration tools, and managing certain network and  diagnostic administration.  See [`calicoctl` directory]({{site.baseurl}}/{{page.version}}/reference/calicoctl) for implementation of calicoctl. |
+
+## Libraries
+
+### [libcalico](https://github.com/projectcalico/libcalico)
+
+| Artifact | Type | Description |
+|---------|-------|-----------|
+| libcalico | pypi library | Contains a variety of helper methods and classes for manipulating the Calico data in an etcd datastore, IP address management and useful namespace utilities to manipulating container interfaces. |
+| `calico/test` | Docker Image | Contains useful shared testing framework code and dependencies for running unit and system tests of Calico in Docker. |
+| `calico/build` | Docker Image | Build image which includes libcalico and the necessary python tooling to produce binaries from python source code. |
+
+## [libcalico-go](https://github.com/projectcalico/libcalico)
+
+| Artifact | Type | Description |
+|---------|-------|-----------|
+| libcalico-go | golang library | Contains a variety of helper methods and classes for manipulating the Calico data in an etcd datastore, IP address management and useful namespace utilities to manipulating container interfaces. |
```

Comment:
Let's not talk about etcd at all here.  We're moving to a world with multiple supported + potential datastores.

libcalico / libcalico-go are just code level interfaces into the Calico API.